ivp/ivp.go

```diff
@@ -0,0 +1,45 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ivp
+
+import (
+	"gonum.org/v1/gonum/mat"
```

Comment:
go imports failed me. should i reset hard again?

Reply:
No, that's fine, it can be cleaned up after discussion. Do you want to open a proposal to add this package?